apps/web/app/(authenticated)/(app)/app/keys/[keyId]/settings/update-key-enabled.tsx

```diff
@@ -23,43 +32,81 @@ type Props = {
 };
 
 export const UpdateKeyEnabled: React.FC<Props> = ({ apiKey }) => {
-  const [enabled, setEnabled] = useState(apiKey.enabled);
+  const router = useRouter();
+  const form = useForm<z.infer<typeof formSchema>>({
+    resolver: zodResolver(formSchema),
+    mode: "all",
+    shouldFocusError: true,
+    delayError: 100,
+    defaultValues: {
+      keyId: apiKey.id ? apiKey.id : undefined,
```

Comment:
according to the Props type above, these can not be undefined

Reply:
Very true my bad.